{{cookiecutter.repo_name}}/.pre-commit-config.yaml

```diff
@@ -1,12 +1,9 @@
 repos:
```

Comment:
I would also suggest to add `pre-commit-hooks`, at least `check-yaml` is very useful to catch silly errors. See what I am using in my app.

https://github.com/ispg-group/aiidalab-ispg/blob/d4ab6ce6972b1be45a6669007cb343c99ecec90c/.pre-commit-config.yaml#L7